vendor/bat-native-ads/src/bat/ads/database.cc

```diff
@@ -213,8 +213,8 @@ mojom::DBCommandResponseInfo::StatusType Database::Migrate(
     return mojom::DBCommandResponseInfo::StatusType::INITIALIZATION_ERROR;
   }
 
-  meta_table_.SetVersionNumber(version);
-  meta_table_.SetCompatibleVersionNumber(compatible_version);
+  CHECK(meta_table_.SetVersionNumber(version));
+  CHECK(meta_table_.SetCompatibleVersionNumber(compatible_version));
```

Comment:
is this a part of a utility process or the main process? We don't want this to be fatal for the main process (browser startup).
cc @brave/ads-dev

Reply:
Fixing to remain as it was before, with no effect. ff31af7173a5cdac98038db77a4dae6a7daf24d2